eureka-client-archaius2/build.gradle

```diff
@@ -1,7 +1,11 @@
 apply plugin: 'nebula.test-jar'
 
+def archaius2Version = '2.1.4'
```

Comment:
Some known issues in 2.1.4. Best to wait for a patch release or go back to 2.0.6